code/game/turfs/simulated/wall/misc_walls.dm

```diff
@@ -7,6 +7,7 @@
 	smooth = SMOOTH_MORE
 	sheet_type = /obj/item/stack/sheet/runed_metal
 	sheet_amount = 1
+	explosion_block = 10
```

Comment:
10???
5 is way better then blocking 10 DEV making bombs 100% useless : /

Reply:
Does 10 completely block explosions? That doesn't seem fair.